src/config.js

```diff
@@ -232,10 +236,16 @@ export default class Config {
       networkConcurrency: this.networkConcurrency,
     });
 
+    this.cacheFolder = String(
```

Comment:
After thinking about this I think this.cacheFolder should contain the folder with version.
Otherwise you will have to modify every command that reads cacheFolder from config